docs/release.md

```diff
@@ -348,9 +348,36 @@ sourced from the delta of PRs merged since the last official release. The text
 from those PRs are sourced directly from the ```release-notes``` section in
 each PRs description.
 
-Below is an example of getting the release notes for v0.31.0-rc.0
+## Using GitHub labels for Release Notes
+
+We use GitHub labels to filter release notes, first using a 'kind/' label and
+second using a 'sig/' label where applicable.
+In some cases, 'kubevirt-bot' applies one or more of these labels automatically,
+however they might need to be manually applied by either the author or a reviewer of the PR.
+
+## The 'kind/' Labels Used for Release Notes
+
+* ```kind/required-change```: The PR adds, removes, or otherwise changes something that is fundamental for a KubeVirt admin to understand before upgrading.
```

Comment:
What do you think of `kind/action-required` instead?

Reply:
Yep, I prefer that.